src/Storages/Statistics/Statistics.h

```diff
@@ -83,6 +87,36 @@ class ColumnsDescription;
 using ColumnStatisticsPtr = std::shared_ptr<ColumnStatistics>;
 using ColumnsStatistics = std::vector<ColumnStatisticsPtr>;
 
+/**
```

Comment:
Cosmetics I: Let's use `///` instead of C-style `/* ... */`.

Cosmetics II: In l. 103, please move each initializing statement to its own line